uint64 timeStamp
uint16 fS
uint16 nTracks
uint64[] ids
float32[] data
